lib/src/DOtherSide.cpp

```diff
@@ -130,8 +130,7 @@ void dos_qapplication_initialize_opengl()
 void dos_qguiapplication_create()
 {
     static int argc = 1;
-    static char empty[1] = {0};
-    static char *argv[] = {empty};
+    static char *argv[] = {"Status"};
```

Comment:
Should this be `"Status Desktop"`?

Reply:
OK, I see from [the status-desktop PR comment](https://github.com/status-im/status-desktop/pull/2953/files#r672265160) that this is meant to represent the executable name. I believe we have two different names, "StatusDev" and "Status", depending on the build env, but I also could have the wrong idea about what this string is meant to represent.

Reply:
Not a name, but full path of the executable.
In our case we just pass any non empty string to not make Qt confused due to an empty argv[0]
In fact, this is not used by Qt, except the case when you did not set applicationName property explicitly, in this case Qt will be using argv[0] as the application name. But we always set that property in mail.qml, so this argv[0] will always be unused.

Reply:
Ah ok, all makes sense now. Thank you! 👍